client/src/js/views/DashboardDogsListDogView.js

```diff
@@ -7,26 +8,38 @@ var DashboardDogsListDogView = Backbone.View.extend({
   template: _.template(DashboardDogsListDogTemplate),
 
   events: {
-    'change [name="checked_in"]': 'handleCheckedInChange',
-    'change [type="radio"]' : 'statusChange'
+    'click .button--check-in': 'handleCheckedInChange',
+    'change [type="radio"]' : 'statusChange',
+    'click .dropdown' : 'toggleDropdown'
   },
 
   initialize: function() {
     this.render();
-  },
 
+    this.listenTo(this.model, 'change:current_status', this.render);
+  },
+ 
   render: function() {
-    this.$el.html(this.template(this.model.attributes));
+    this.$el.html(this.template(this.model.toJSON()));
     return this;
   },
 
   handleCheckedInChange: function(e) {
-    this.model.set('checked_in', e.target.checked);
+    this.model.checkIn();
```

Comment:
I understand what you're trying to do here but I think you are jumping the gun on getting dog's to be checked in :stuck_out_tongue:. Check ins will be handled solely by the server since we only want to report the check ins from the hydrants. In other words, if a dog is OOO, a user should not be allowed to check that dog in.